reactioncommerce/reaction
# refactor: move getPaymentMethodConfigByName to context

Resolves #5653
Impact: **minor**  
Type: **feature|refactor**

## Issue
`getPaymentMethodConfigByName` was used across many plugins

## Solution
For the sake of keeping our plugins contained, we are adding this method, `getPaymentMethodConfigByName` to the `context` to use instead of importing from the `payments` package directly.

## Breaking changes
Any custom code which uses `getPaymentMethodConfigByName` by direct import will need to update to use the `context.queries.getPaymentMethodConfigByName` instead

## Testing
1. Start the app
1. See you can create an order
